packages/calcite-components/src/components/tree-item/tree-item.e2e.ts

```diff
@@ -450,6 +450,47 @@ describe("calcite-tree-item", () => {
     expect(collapseSpy).toHaveReceivedEventTimes(1);
   });
 
+  describe("selection event", () => {
+    let page: E2EPage;
+    let treeItem: E2EElement;
+    let selectSpy: EventSpy;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(html`
+        <calcite-tree>
+          <calcite-tree-item id="item-1">Item 1</calcite-tree-item>
+        </calcite-tree>
+      `);
+
+      treeItem = await page.find("#item-1");
+      selectSpy = await treeItem.spyOnEvent("calciteTreeItemSelect");
+      await page.waitForChanges();
```

Comment:
This call shouldn't be necessary. `setContent` calls it behind the scenes and the above calls don't need to be waited on.